apps/testrunner/ssh.py

```diff
@@ -2,7 +2,7 @@
 import os
 import socket
 import tempfile
-from six import StringIO
+from six import BytesIO
```

Comment:
The only difference is that it works for Python 2.x and Python 3.x when you deal with unicode strings. StringIO doesn't work well in Pythin 3.x and data received from channel.